modules/assim.batch/R/pda.emulator.R

```diff
@@ -404,10 +407,14 @@ pda.emulator <- function(settings, params.id = NULL, param.names = NULL, prior.i
   # prepare for parallelization
   dcores <- parallel::detectCores() - 1
   ncores <- min(max(dcores, 1), settings$assim.batch$chain)
-  cl <- parallel::makeCluster(ncores, type="FORK")
+  
+  logger.setOutputFile(paste0(settings$outdir, "/pda.log"))
+  
```

Comment:
I tried something like this @ashiklom @mdietze @robkooper 

It writes out to an outfile but leaving it empty didn't make it print-out to the console. It's too messy to be printed out to the console anyway, writes out both the jump variances and the number of iterations at each 100th step (I set it to 200 now in the code). But I think both master and the workers are writing to the file somehow because messages are being written twice per worker or I missed something. 

[pda.txt](https://github.com/PecanProject/pecan/files/627854/pda.txt)